images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -137,424 +140,434 @@ function Get-NVMNodeVersionList {
     $nodejsVersionsRaw = Run-Command "${nvmInitCommand} && nvm ls"
     $nodeVersions = $nodejsVersionsRaw | ForEach-Object { $_.TrimStart(" ").TrimEnd(" *") } | Where-Object { $_.StartsWith("v") }
     $result = [string]::Join(" ", $nodeVersions)
-    return "NVM - Cached node versions: $result"
+    return $result
 }
 
 function Build-OSInfoSection {
+    param (
+        [string] $ImageName
+    )
+
     $fieldsToInclude = @("System Version:", "Kernel Version:")
     $rawSystemInfo = Invoke-Expression "system_profiler SPSoftwareDataType"
     $parsedSystemInfo = $rawSystemInfo | Where-Object { -not ($_ | Select-String -NotMatch $fieldsToInclude) } | ForEach-Object { $_.Trim() }
-    $output = ""
     $parsedSystemInfo[0] -match "System Version: macOS (?<version>\d+\.\d+)" | Out-Null
     $version = $Matches.Version
-    $output += New-MDHeader "macOS $version info" -Level 1
-    $output += New-MDList -Style Unordered -Lines $parsedSystemInfo -NoNewLine
-    return $output
+    $systemVersion = $parsedSystemInfo[0].Replace($fieldsToInclude[0],"").Trim()
+    $kernelVersion = $parsedSystemInfo[1].Replace($fieldsToInclude[1],"").Trim()
```

Comment:
nit: use `TrimStart` instead of `Replace`.

Reply:
we found some problems with TirmStart, so changing approach here :)